README.md

```diff
@@ -122,6 +123,11 @@ Do you like to talk?
 #### Identity Verification
 * BlockScore | https://blockscore.com | [@getblockscore](https://twitter.com/getblockscore) | $1/verification | Individual ID verifications and company verifications | Verify customers in realtime. Implement in hours.
 
+
+
+#### International Mobile Number Verification
+* Cognalys,Inc | https://www.cognalys.com | [@cognalys](https://twitter.com/cognalys) | $12 /mo 1200 Tries per day | Freemium mobile number verification through an innovative and relaible  method than using SMS gateway. Free accounts will have 70 Tries and 50 verifications per day .Both API and android SDK's are available .
```

Comment:
Cognalys,Inc -> Cognalys. There's also a couple of double-spaces and misplaced full-stops.